aws/data_source_aws_rds_orderable_db_instance.go

```diff
@@ -195,6 +203,24 @@ func dataSourceAwsRdsOrderableDbInstanceRead(d *schema.ResourceData, meta interf
 		input.EngineVersion = aws.String(v.(string))
 	}
 
+	if v, ok := d.GetOk("default_only"); ok && v.(bool) {
```

Comment:
`d.GetOk()` automatically checks against the zero-value for the type (would also recommend alphabetizing this due to the next comment)

```suggestion
	if v, ok := d.GetOk("default_only"); ok {
```

Reply:
Getting rid of `default_only` makes this OBE.